sympy/stats/symbolic_probability.py

```diff
@@ -89,11 +90,11 @@ class Expectation(Expr):
     >>> Expectation(X + Y)
     Expectation(X + Y)
 
-    To expand the ``Expectation`` into its expression, use ``doit()``:
+    To expand the ``Expectation`` into its expression, use ``expand()``:
 
-    >>> Expectation(X + Y).doit()
+    >>> Expectation(X + Y).expand()
     Expectation(X) + Expectation(Y)
-    >>> Expectation(a*X + Y).doit()
+    >>> Expectation(a*X + Y).expand()
```

Comment:
Can you please make a separate PR for `expand` fix if these aren't related with the changes in `doit`? It will be easier to trace in future.

Reply:
I have made it together because it will lead to some test failures, so I tried fixing them together.

Reply:
I have added this change in the release notes to be highlighted

Reply:
I will open PR for renaming